file.txt

```diff
@@ -1 +1,2 @@
 Mohandas Karamchand Gandhi was an Indian lawyer, anti-colonial nationalist, and political ethicist, who employed nonviolent resistance to lead the successful 
+The same year Gandhi adopted the Indian loincloth, or short dhoti and, in the winter, a shawl, both woven with yarn hand-spun on a traditional Indian spinning wheel, or charkha, as a mark of identification with India's rural poor. 
```

Comment:
the line which you added is not the correct information plz review and change the line

Reply:
ok i will do